data/modelCuration/v8_6_0/newGPRsfromDBs.tsv

```diff
@@ -0,0 +1,20 @@
+rxnID_yeast_model	genes_yeast_model	final_GPR	Uniport	SGD
+r_1614	NAN	(YGL101W or  YBR242W)	https://www.uniprot.org/uniprot/P53144; https://www.uniprot.org/uniprot/P38331	https://www.yeastgenome.org/locus/S000000446; https://www.yeastgenome.org/locus/S000003069
+r_1615	NAN	(YGL101W or  YBR242W)	https://www.uniprot.org/uniprot/P53144; https://www.uniprot.org/uniprot/P38331	https://www.yeastgenome.org/locus/S000000446; https://www.yeastgenome.org/locus/S000003069
+r_1616	NAN	(YGL101W or  YBR242W)	https://www.uniprot.org/uniprot/P53144; https://www.uniprot.org/uniprot/P38331	https://www.yeastgenome.org/locus/S000000446; https://www.yeastgenome.org/locus/S000003069
+r_1617	NAN	(YGL101W or  YBR242W)	https://www.uniprot.org/uniprot/P53144; https://www.uniprot.org/uniprot/P38331	https://www.yeastgenome.org/locus/S000000446; https://www.yeastgenome.org/locus/S000003069
+r_4584	NAN	(YOR317W or YER015W or YMR246W or YIL009W)	https://www.uniprot.org/uniprot/P30624; https://www.uniprot.org/uniprot/P39518; https://www.uniprot.org/uniprot/P47912; https://www.uniprot.org/uniprot/P39002	https://www.yeastgenome.org/locus/S000005844; https://www.yeastgenome.org/locus/S000000817; https://www.yeastgenome.org/locus/S000004860; https://www.yeastgenome.org/locus/S000001271
+r_4372	NAN	YGR247W	https://legacy.uniprot.org/uniprot/P53314	https://www.yeastgenome.org/locus/S000003479
+r_4378	NAN	(YBR299W or YGR292W)	https://legacy.uniprot.org/uniprot/P38158; https://legacy.uniprot.org/uniprot/P53341	https://www.yeastgenome.org/locus/S000000503; https://www.yeastgenome.org/locus/S000003524
+r_4388	NAN	YGR247W	https://legacy.uniprot.org/uniprot/P53314	https://www.yeastgenome.org/locus/S000003479
+r_4400	NAN	(YBR299W or  YGR292W)	https://legacy.uniprot.org/uniprot/P38158; https://legacy.uniprot.org/uniprot/P53341	https://www.yeastgenome.org/locus/S000000503; https://www.yeastgenome.org/locus/S000003524
+r_4412	NAN	YGR247W	https://legacy.uniprot.org/uniprot/P53314	https://www.yeastgenome.org/locus/S000003479
+r_4420	NAN	(YBR299W or  YGR292W)	https://legacy.uniprot.org/uniprot/P38158; https://legacy.uniprot.org/uniprot/P53341	https://www.yeastgenome.org/locus/S000000503; https://www.yeastgenome.org/locus/S000003524
```

Comment:
Please double-check these, the gene names, Uniprot and SGD links do not completely match, `YBR299W` is a maltase.

Reply:
I recheck these, (YBR299W or YGR292W) can not catalyze r_4378, but could catalyze maltotriose(r_4400) and  turanose(r_4420). 
In terms of the links, I didn't find problems.
| gene name| geneShortName | SGD | uniprot |
| --- | --- | --- | --- |
| YBR299W | MAL32 | https://www.yeastgenome.org/locus/S000000503 | https://legacy.uniprot.org/uniprot/P38158 |
| YGR292W | MAL12 | https://www.yeastgenome.org/locus/S000003524 | https://legacy.uniprot.org/uniprot/P53341 |

Reply:
I got confused because `YGR247W` in reaction `r_4372` got the same Uniprot link as a gene in `r_4378`?

Reply:
You're absolute right, my mistake!